Gemfile

```diff
@@ -86,7 +86,7 @@ gem "inifile",                        "~>3.0",     :require => false
 gem "logging",                        "~>1.8",     :require => false  # Ziya depends on this
 gem "net_app_manageability",          ">=0.1.0",   :require => false
 gem "net-ping",                       "~>1.7.4",   :require => false
-gem "net-ssh",                        "~>3.2.0.rc2",   :require => false
+gem "net-ssh",                        "=3.2.0",    :require => false
```

Comment:
We might want to do "~> 3.2.0" to pull in patch updates in the future.  This is good for backporting to darga after the first release.